src/components/Integrations/Shopify/Shopify.js

```diff
@@ -99,6 +81,31 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  const getData = useCallback(async () => {
```

Comment:
Why is _useCallback_ necessary?

Reply:
If I use the function without useCallback I get this warning:
```  Line 84:  The 'getData' function makes the dependencies of useEffect Hook (at line 103) change on every render. To fix this, wrap the 'getData' definition into its own useCallback() Hook  react-hooks/exhaustive-deps ```
Since I use this in two places at the same time I've wrapped this up into useCallback hook.
https://github.com/facebook/react/issues/14920

Reply:
`useCallback` was recommended in the warning, but it was not a good idea, because it memoizes the function. It is fixed since we do not need `getData` more than a time.